src/main/java/com/forgeessentials/scripting/ScriptMethods.java

```diff
@@ -271,13 +271,13 @@ public String getHelp()
 
     protected static boolean getPermcheckResult(ICommandSender sender, String[] args)
     {
-        if (!(sender instanceof EntityPlayerMP))
-            if (sender instanceof MinecraftServer)
```

Comment:
I changed it to get the server ident so it will be more robust and work with other server like senders that are not instances of MinecraftServer.

Now it will fetch the permission nodes related to that non player entity.